src/libs/antares/study/constraint/constraint.hxx

```diff
@@ -114,6 +114,11 @@ inline BindingConstraint::const_iterator BindingConstraint::end() const
     return pLinkWeights.end();
 }
 
+inline bool BindingConstraint::skipped() const
+{
+    return !(linkCount() > 0 || enabledClusterCount() > 0);
```

Comment:
`linkCount() == 0 && enabledClusterCount() == 0`

Reply:
This comment to tell gitHub I have requested changes (the same as those spotted by flomnes)